flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/codegen/CodeGeneratorContext.scala

```diff
@@ -765,4 +765,9 @@ object CodeGeneratorContext {
   def apply(config: TableConfig): CodeGeneratorContext = {
     new CodeGeneratorContext(config)
   }
+
+  val DEFAULT_OUT_RECORD_TERM = "out"
```

Comment:
remove the two and use `CodeGenUtils. DEFAULT_XXX`

Reply:
done